scenarios/01_Breaking_the_Circle.cfg

```diff
@@ -672,19 +674,66 @@
                     x=16
                     y=4
                     side=2
-                    # Oh well, I tried:
+                    # Oh well, it was worth the try:
                     profile=portraits/Medic.png
                 [/unit]
                 [scroll_to_unit]
                     id=Medic
                 [/scroll_to_unit]
                 [redraw][/redraw]
                 {MSG_Reme _"My lady, we have found a medic!"}
+                [scroll_to_unit]
+                    id=Medic
+                    highlight=yes
+                [/scroll_to_unit]
+                [redraw][/redraw]
+                [delay]
+                    time=234
+                [/delay]
                 {MSG_Lorin _"Underling! My son is dying. If you heal him, I will spare your life. If not, I will burn you and your family alive, do you understand?"}
                 # TODO: this medic should have a (better) older-looking portrait, to match this dialogue:
                 {MESSAGE (Medic) () (_"Medic") _"Don't try to intimidate me. I am too old to be afraid of death, I have no family. I will try to heal your son because of my vows as a healer, not because of your threats, Akladian lady."}
                 {MSG_Lorin _"Your motives are of no concern to me, just do it!"}
+                {MESSAGE (Medic) () (_"Medic") _"Hold on a moment."}
+                [move_unit]
+                    id=Medic
+                    to_x=15,16
+                    to_y=5,4
+                [/move_unit]
+                [store_unit]
+                    [filter]
+                        id=Medic
+                    [/filter]
+                    variable=uMedic
+                [/store_unit]
+                [modify_unit]
+                    [filter]
+                        id=Medic
+                    [/filter]
+                    facing=$(relative_dir(loc($uMedic.x,$uMedic.y), loc(16,4)))
+                [/modify_unit]
+                [heal_unit]
+                    [filter]
+                        id=Gawen Hagarthen
+                    [/filter]
+                    [filter_second]
+                        id=Medic
+                    [/filter_second]
+                    animate=yes
+                [/heal_unit]
                 {CLEAR_VARIABLE ano_gawen_poisoned}
+                [delay]
+                    time=234
+                [/delay]
+                [redraw][/redraw]
+                [delay]
+                    time=234
+                [/delay]
+                {MESSAGE (Medic) () (_"Medic") _"...and there you go."}
```

Comment:
Probably doesn't need a message.

Reply:
I think it helps to mark the end of the animation, and to give another chance to see the portrait

Reply:
...I'll add a comment to that effect.

Reply:
Comment added in [8029e28](https://github.com/nemaara/A_New_Order/pull/171/commits/8029e2809a6f79ad9dee5da1030af349783d9dc6).